pom.xml

```diff
@@ -80,6 +80,11 @@
             <artifactId>liquibase-core</artifactId>
             <version>4.5.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
```

Comment:
I feel like I asked this earlier? But don't see it in the discussion...

Adding httpclient as a dependency makes it more difficult for people to use the extension since they also need to add that to the lib dir or their classpath. And, in embedded setups you can get into version conflicts as well.

It is a pain to have to implement functionality with the lower-level java.net libraries, but how terrible would it be in this case?

Reply:
right removed it, it was simple enough to implement custom method